Component/Component/Button/DualButton/FamiliDualButton.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  FamiliDualButton.swift
+//  Component
+//
+//  Created by William Inx on 12/06/20.
+//  Copyright © 2020 William Inx. All rights reserved.
+//
+
+import UIKit
+
+/** Famili Dual Button Template
+
+Create two buttons with predefined style.
+How to use:
+ * Using XIB
+    * Drag and drop a UIView  and change the class to FamiliDualButton, put it inside a container view
+ * Programatically
+    * Init using the custom initializer to set all the required value
+*/
+
+public protocol FamiliDualButtonDelegate: class {
+    func didTapLeftButton(sender: UIButton)
+    func didTapRightButton(sender: UIButton)
+}
+
+@IBDesignable public class FamiliDualButton: UIView {
+    // MARK: - Designable
+    
+    typealias ButtonProperty = FamiliDualButtonConstant.CommonProperties
```

Comment:
please update typealias name to be more meaningful (since it now hold more than buttonProperty) perhaps CommonProperty?

Reply:
okay noted